ote/src/cljs/ote/views/service_search.cljs

```diff
@@ -273,8 +274,26 @@
 
 (def transport-types [:road :rail :sea :aviation])
 
+(defn- operation-area-searchbox [e! operation-area-filter-completions]
+  (let [suggestions (mapv (fn [s] {:text (format/postal-code-at-end s) :value s}) operation-area-filter-completions)]
+    {:name ::t-service/operation-area
+     :type :chip-input
+     :container-class "col-xs-12 col-sm-4 col-md-4"
+     :hint-text (tr [:service-search :operation-area-search-placeholder])
+     :filter (constantly true)
+     :hint-style {:top "20px"}
```

Comment:
Wouldn't `rem` work better in case user changes the text size in browser (ctrl/cmd+ or -)

Reply:
Yes